alembic/operations/ops.py

```diff
@@ -1091,10 +1094,18 @@ def create_table(cls, operations, table_name, *columns, **kw):
 class DropTableOp(MigrateOperation):
     """Represent a drop table operation."""
 
-    def __init__(self, table_name, schema=None, table_kw=None, _reverse=None):
+    def __init__(self,
+                 table_name,
+                 schema=None,
+                 table_comment=None,
```

Comment:
**mike bayer** (zzzeek) wrote:

I think these both need to come from table_kw using the "pop" form.  if we were to refactor this further, "table_kw" would become "**kw" just like CreateTableOp.

Also we need to add "prefixes" here.

View this in Gerrit at https://gerrit.sqlalchemy.org/c/sqlalchemy/alembic/+/2989